content/blog/2018/data-plane-setup/index.md

```diff
@@ -0,0 +1,533 @@
+---
+title: Demystifying Istio's Sidecar Injection Model 
+description: De-mystify how Istio manages to plugin its data-plane components into an existing deployment. Deep-dive into sidecar injection, how it is done whether manually or `namespaceSelector` based automatic injection. And also how the iptables are modified in the pod namespace to redirect the traffic to the injected proxy. 
```

Comment:
Please keep descriptions to a single sentence. This should be plain text, no markdown.

Reply:
Done.